include/git2/sys/filter.h

```diff
@@ -197,6 +197,9 @@ typedef int GIT_CALLBACK(git_filter_apply_fn)(
 	const git_buf *from,
 	const git_filter_source *src);
 
+/**
+ * Callback used for filter streaming
+ */
```

Comment:
I guess there's no parameter descriptions here as you weren't able to quite make sense of it? Fair enough if that's the case, I'm not able to make much sense of it either without diving into code.

Reply:
Exactly. I'm not sure what's the correct way forward though: this will silence the docurium warning, since it's now documented. But the "cleverer" checks are made via `-Wdocumentation`, and by design it ignores missing docblocks/directives (hence no missing `@param` arg).

Reply:
I've dropped that comment, for now. Let's wait for some pressure to investigate to show up 😉.